### PR TITLE
Adding option to skip fixRef

### PR DESCRIPTION
**Pull request by jasonh-n-austin**
There are many cases when external schema files are used, where using `fixRef` is inappropriate (and mangles `$ref` pointing to external schema files.
Swagger 2 supports external refs (at least in the spec, if not mixed in lib support). See [swagger-parser](https://github.com/BigstickCarpet/swagger-parser) for a great example.

* Allows for `convert()` to accept a second optional param, `options`.
* `options.refsToDefinitions: true` will run `fixRef` on all references (default: true). Backward compatible with current interface.
* `options.refsToDefinitions: false` leaves `$ref`s as-is.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -16,22 +20,33 @@ exports.getVersion = function (data) {
   return data.discoveryVersion;
 };
 
-exports.convert = function (data) {
+/**
```
Comment (jasonh-n-austin) [inline src/index.js]: Also added instructions in code for `convert` method.

**Comment by IvanGoncharov** [comment]
@jasonh-n-austin Thank you for PR.
I never expect anyone outside of Google to use this format, so I based my converter solely on Google specs. Can you make example spec with external refs and I will add it to test suite?

Can external refs be automatically detected? I don't like idea of global switch and if possible prefer to detect them. Can presence of `/` symbol be such criteria?

**Comment by IvanGoncharov** [comment]
I tested solution with slash detection and it doesn't brake compatibility with Google specs.
Can you test following path
```diff
diff --git a/src/index.js b/src/index.js
index 076006d..6a263de 100644
--- a/src/index.js
+++ b/src/index.js
@@ -94,6 +94,8 @@ function processGlobalParameters(parameters, srGlobalRefParameters) {
 }
 
 function fixRef(ref) {
+  if (ref.indexOf('/') !== -1)
+    return ref;
   return '#/definitions/' + ref;
 }
 
```

**Comment by IvanGoncharov** [comment]
@jasonh-n-austin Did you have a chance to test my patch?

**Comment by jasonh-n-austin** [comment]
Sorry didn't get the GH notifications on this. 
I'll take a look at it by the end of the week. I hadn't realized tests were breaking, I just pushed this up on my way out the door on an "open dev" day.
Yes, shockingly there are a handful of things using GDD still. Trying to help move them forward :)

**Comment by IvanGoncharov** [comment]
@jasonh-n-austin Thanks.
P.S. Build was broken before your changes, I just fixed it on weekends.

**Comment by jasonh-n-austin** [comment]
Slash detection wouldn't help here. If the schema file is in the same directory, there's nothing leading. 
Example:
```
"request": {
  "$ref":"resource.json"
},
```

Thinking about it more: if we detect if `$ref`'s value contains `.json`, that would probably do the trick. 
Closing this, will open another PR.

**Pull request closed without merging**